@rishabh1024 

Hey,

I've run CI workflows and it looks like MyPy one has an error. Do you think you will have time to check and fix it?

```
mypy: commands[0]> mypy icecream
icecream/icecream.py:567: error: Function is missing a return type annotation  [no-untyped-def]
icecream/icecream.py:569: error: Function is missing a return type annotation  [no-untyped-def]
icecream/icecream.py:583: error: Function is missing a type annotation  [no-untyped-def]
icecream/icecream.py:586: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
Found 3 errors in 1 file (checked 5 source files)
mypy: exit 1 (1.70 seconds) /home/runner/work/icecream/icecream> mypy icecream pid=2267
  mypy: FAIL code 1 (11.26=setup[9.56]+cmd[1.70] seconds)
  evaluation failed :( (11.29 seconds)
  ```

--- a/icecream/icecream.py
+++ b/icecream/icecream.py
@@ -17,7 +17,7 @@ import inspect
 import pprint
 import sys
 import time
-from types import FrameType
+from types import FrameType, TracebackType
 from typing import (
     Optional,
     cast,
@@ -564,9 +564,9 @@ class Timer:
             msg = f"{formatted_time}"
         self._ic.outputFunction(msg)
 
-    def __call__(self, func: Callable[..., Any]):
+    def __call__(self, func: Callable[..., Any]) -> Callable[..., Any]:
         @functools.wraps(func)
-        def wrapper(*args: Any, **kwargs: Any):
+        def wrapper(*args: Any, **kwargs: Any) -> Any:
             start_time: float = time.perf_counter()
             try:
                 return func(*args, **kwargs)
@@ -580,7 +580,7 @@ class Timer:
         self._enter_time = time.perf_counter()
         return self
 
-    def __exit__(self, exc_type, exc_value, traceback):
+    def __exit__(self, exc_type: Optional[Type[BaseException]], exc_value: Optional[BaseException], traceback: Optional[TracebackType]) -> None:
         if self._enter_time is None:
             raise RuntimeError("Timer.__exit__ called without __enter__. ")
         duration: float = time.perf_counter() - self._enter_time

